src/waveform/widgets/waveformwidgettype.h

```diff
@@ -6,16 +6,26 @@ class WaveformWidgetType {
         // The order must not be changed because the waveforms are referenced
         // from the sorted preferences by a number.
         EmptyWaveform = 0,
-        SoftwareSimpleWaveform,  //TODO
-        SoftwareWaveform,        // Filtered
-        QtSimpleWaveform,        // Simple Qt
-        QtWaveform,              // Filtered Qt
-        GLSimpleWaveform,        // Simple GL
-        GLFilteredWaveform,      // Filtered GL
-        GLSLFilteredWaveform,    // Filtered GLSL
-        HSVWaveform,             // HSV
-        GLVSyncTest,             // VSync GL
-        RGBWaveform,             // RGB
+        SoftwareSimpleWaveform, //TODO
+#ifndef __APPLE__
```

Comment:
This would again alter the numbering, even if only on macOS. Not permitted.

The enumeration must not be modified, even if entries are unused. Filtering must be performed elsewhere.

Reply:
it's not working anyway..